gluetool/action.py

```diff
@@ -81,11 +81,11 @@
 
 DEFAULT_TRACING_SERVICE_NAME = 'gluetool'
 DEFAULT_TRACING_REPORTING_HOST = '127.0.0.1'
-DEFAULT_TRACING_REPORTING_PORT = 5775
-DEFAULT_TRACING_FLUSH_TIMEOUT = 30
+DEFAULT_TRACING_REPORTING_PORT = '5775'
+DEFAULT_TRACING_FLUSH_TIMEOUT = '30'
 
 
-class Tracer(object):
+class Tracer():
```

Comment:
Why this change? It won't work in Python 2.7 anymore, making `Tracer` an old-style class.

Reply:
a leftover from experiments :/